### DISCUSSION TITLE
Initial Installation Failed - Apache & Nextcloud containers stuck in "starting" status

### DISCUSSION THREAD

**Discussion by Lwfrancisco** [nextcloud/all-in-one#770] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Install docker
2. Configured nginx-proxy-manager to point to the AIO instead of nextcloudpi on another system.
3. Run startup command here: https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md

### Expected behavior 
Expect all containers to start.

### Actual behavior 
Apache and Nextcloud Containers stuck in loop.

### Host OS 
Debian 11 (Under LXC/proxmox)

#### Nextcloud AIO version 
latest

#### Other valuable info 
logs for apache container (loops):
```
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9000: Name or service not known
Waiting for Nextcloud to start...
```

logs for nextcloud container (loops):
```
psql: error: connection to server at "nextcloud-aio-database" (172.18.0.5), port 5432 failed: FATAL:  password authentication failed for user "oc_nextcloud"
Configuring Redis as session handler
Appdata is not present. Did you maybe change the datadir after aio was first started?
```

**Comment by szaimen** [comment, marked as answer]
This seems to be a hickup. Please reset the instance https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance and try again.

**Reply by Lwfrancisco** [reply]
Oddly enough, I'd tried doing a new LXC container but had the same result. But this ended up working.

Thanks!
